src/rafimage.cpp

```diff
@@ -29,14 +29,22 @@ std::string RafImage::mimeType() const {
 }
 
 uint32_t RafImage::pixelWidth() const {
-  auto widthIter = exifData_.findKey(Exiv2::ExifKey("Exif.Photo.PixelXDimension"));
+  // Try first the uncropped sensor width from FujiIFD
+  auto widthIter = exifData_.findKey(Exiv2::ExifKey("Exif.Fujifilm.RawImageFullWidth"));
+  if (widthIter != exifData_.end() && widthIter->count() > 0)
```

Comment:
is there an empty method?

Reply:
Changed the PR considerably for a different behaviour now. @victoryforce